packages/dataviews/src/add-filter.js

```diff
@@ -21,6 +21,7 @@ const {
 	DropdownMenuGroupV2Ariakit: DropdownMenuGroup,
 	DropdownMenuItemV2Ariakit: DropdownMenuItem,
 	DropdownMenuSeparatorV2Ariakit: DropdownMenuSeparator,
+	DropdownMenuItemLabelV2Ariakit: DropdownMenuItemLabel,
```

Comment:
This is weird as a component, why not a prop of `DropdownMenuItem` ?

Reply:
`DropdownMenuItemLabel` (together witn `DropdownMenuItemHelpText`) are useful to use the correct typography, and to make sure that the text gets truncated appropriately (label after one line, help text after two lines).

The reason why I exposed them as separate components is because I wanted to give consumers of `DropdownMenu` the freedom of passing any React component as `children`. Folks can opt in to the "default" look by using `DropdownMenuItemLabel` and/or DropdownMenuItemHelpText`, but they can also pass any other content in case they want to go custom.

Using dedicated props instead of `children` would either limit the flexibility of what we can display in the menu item (ie. what if folks don't want to truncate the label text after 1 line?), or even worse, end up in a similar situation to `Button` where we currently have a `text` prop _and_ a `children` prop.

Having said that, the component is still under active development and only exposed via private APIs, so we can make changes as we see fit.